examples/zephyr/hello_nrf91_offloaded/README.md

```diff
@@ -49,8 +49,16 @@ AT%CMNG=3,515765868,3
 Execute following AT commands over serial to provision new credentials:
 
 ``` sh
-AT%CMNG=0,515765868,4,my-psk-id
-AT%CMNG=0,515765868,3,my-psk-encoded-as-hexstring
+AT%CMNG=0,515765868,4,"my-psk-id"
```

Comment:
@szczys when issuing the AT commands via `nrf_modem_at` below should the PSK / PSK ID also be surrounded by `""`?

Reply:
In my testing, yes they should be. Without them the modem returns ERROR. The [Nordic AT command reference](https://www.nordicsemi.com/-/media/Restricted/ProductKeys/L984675KPNS/nrf91atcommandsv071.pdf) shows double quotes wrapping the credentials (page 45)

Reply:
I  retested this and it does indeed work without double-quotes. I've removed them, thanks for raising the issue.